gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/api/ServiceManager.java

```diff
@@ -58,7 +59,7 @@
 	
 	String startService(String service_id, String runtimeOptions);  //may also need input/output topics or simulation id
 	
-	String startServiceForSimultion(String service_id, String runtimeOptions, String simulationId);  //may also need input/output topics??
+	String startServiceForSimultion(String service_id, String runtimeOptions, Map<String,Object> simulationContext);  //may also need input/output topics??
```

Comment:
Should use something other than Object here...There's a SimulationObject dto or something right?